src/components/MaterialFilter.tsx

```diff
@@ -114,7 +120,7 @@ const MaterialFilter: React.FC = () => {
                   placeholder="Secondary Material"
```

Comment:
It'd probably be helpful to remove the placeholder text here for accessibility purposes. We could take care of that later though.

Reply:
Good call! I'll do that now.